src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,18 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
```

Comment:
`CommentList `should only render list of comments. So it's much better to move `comments.length ? <></> : <></>` to `PostInfo` component